src/components/App/App.js

```diff
@@ -17,7 +17,8 @@ import MenuPage from '../MenuPage';
 import RestaurantsList from '../RestaurantsList/RestaurantsList';
 import RestaurantAdminPage from '../RestaurantAdminPage/RestaurantAdminPage';
 import RestaurantView from '../RestaurantView/RestaurantView';
-import Geolocation from '../geolocation/geolocation'
```

Comment:
I don't know why it shows Geolocation in red. I did not delete this import just FYI